apps/blockchain/scripts/sync_with_infura.ex

```diff
@@ -153,4 +144,14 @@ tree =
       Blockchain.Blocktree.new_tree()
   end
 
+current_block =
```

Comment:
If you wanna go one step further:

It looks like we're only using `current_block` to get the block number on line`157`

So I think we could actually do:

    current_block_number = if is_nil(tree.best_block), do: 0, else: tree.best_block.header.number
    SyncWithInfura.add_block_to_tree(db, chain, tree, current_block_number)

Reply:
Oh yeah, that is a good. I'll add that to my dao fork changes